tests/AssertAnnotationTest.php

```diff
@@ -1509,6 +1509,16 @@ function foo(array $foos) : array {
                      */
                     function allIsInstanceOf($value, $class): void {}'
             ],
+            'implicitReflectionPropertyAssertion' => [
+                '<?php
+                    $class = new ReflectionClass(stdClass::class);
+                    $properties = $class->getProperties();
+                    foreach ($properties as $property) {
+                        if ($property->hasType()) {
+                            $property->getType()->allowsNull();
+                        }
+                    }',
+            ],
```

Comment:
This could/should work:

```suggestion, 
                 [], 
                 [], 
                 '7.4',
            ],
```